### DISCUSSION TITLE
How to http-cache translations files more aggressively?

### DISCUSSION THREAD

**Discussion by Xiphe** [sergiodxa/remix-i18next#73] [General]
Hey, there

Ideally I would want my translation files to have a content hash and then be cached forever as immutable. 

I have a [prove of concept](https://github.com/Xiphe/remix-i18n-testyard/tree/t9n-compiler) where I got this to work by introducing an additional compiler + manifests etc. But am wondering if there is a simpler solution for this.

Have you managed to implement a good http caching strategy for you translation files? Or would you be interested in bringing a solution to this package?

**Comment by sergiodxa** [comment]
I think that's outside the scope of this package. Because you could use a backend to fetch the files from your own app or read from a service like Locize, you could even load all translations at once in localStorage and read from there.

What you describe is responsibility of the backend your setup to load your translations.

**Reply by Xiphe** [reply]
Thanks for the quick answer! I'd personally argue that since remix is both backend and frontend and this package directly targets remix. That concerns of "the backend" might also be concerns of this package. But I can totally see your point and am fine with rolling a custom solution 👍

**Reply by sergiodxa** [reply]
I think loading hashed translations is something any i18next project could benefit, if you have a React SPA you could totally use that backend too.
